hydroflow_lang/src/graph/serde_graph.rs

```diff
@@ -135,11 +142,11 @@ impl SerdeGraph {
         ) -> std::fmt::Result {
             writeln!(
                 write,
-                "{:t$}subgraph \"sg_{:?} stratum {:?}\"",
+                "{:t$}subgraph sg_{sg:?} [\"sg_{sg:?} stratum {:?}\"]",
```

Comment:
did I have those printing backward?

Reply:
No, just providing a separate shorter ID for the subgraphs (`sg_#`) vs their label (`sg_# stratum #`)